src/components/ImageView/index.spec.tsx

```diff
@@ -87,4 +88,9 @@ describe(__filename, () => {
     expect(svg).toContain('<circle');
     expect(svg).not.toContain('<script>');
   });
+
+  it('renders responsive images', () => {
+    const root = render();
+    expect(root.find('img')).toHaveClassName(styles.responsive);
```

Comment:
Yeah, I don't think it's necessary to add a test here because `className={styles.responsive}` is not set based on any props or other inputs so we can trust that it won't change. However, this is just a minor detail. I'd like to merge this in and keep it moving!